### ISSUE TITLE
Do Not Render Markdown in User Input

### ISSUE THREAD

**Issue by xshadow0x** [Deeptrain-Community/chatnio#140]
<img width="100" alt="image" src="https://github.com/Deeptrain-Community/chatnio/assets/106351155/7e745708-2489-4a05-babd-d902b2993bbb">
<img width="57" alt="image" src="https://github.com/Deeptrain-Community/chatnio/assets/106351155/cfd7227c-e934-4511-9524-c32f94b541cd">
<img width="89" alt="image" src="https://github.com/Deeptrain-Community/chatnio/assets/106351155/afed06ca-8de3-4304-a638-564df7e4183b">

bug with "\*" symbol 

there also bug when i send my python code with comments markdown start working and its looking not too good

<img width="90" alt="image" src="https://github.com/Deeptrain-Community/chatnio/assets/106351155/866fd914-fcd6-4529-856d-5290a875e39b">
<img width="142" alt="image" src="https://github.com/Deeptrain-Community/chatnio/assets/106351155/25937279-f444-444e-ab94-6929fff3ee1f">

maybe need to delete some markdown features from user's messages

**Comment by zmh-program** [comment]
Oh, I see.
Now there is indeed a hard problem to solve, markdown is misrecognized in some regular cases. It affects the usage.
I'm thinking about how to solve this kind of problem, maybe we can discuss and share our thoughts?

The way I came up with it is that the `user` role information is not rendered as markdown, but as full text. But it can also be unfriendly in some cases
Or not processed, wrapping 
```
[code]
```

Something like that, but it would interfere with usage in some regular cases.

If there really isn't a better good idea, I'll consider `user` role not rendering markdown later on.

I found this at [ChatGPT-Next-Web](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web) as well, and it does seem to be a bit more tricky:
![image](https://github.com/Deeptrain-Community/chatnio/assets/112773885/5fa0f126-b04d-4117-ba91-38f97d1e29d1)
